components/wso2is7.key.manager/src/main/java/org/wso2/is7/tenant/management/ISTenantManagementRestClient.java

```diff
@@ -0,0 +1,241 @@
+package org.wso2.is7.tenant.management;
```

Comment:
license header

Reply:
I have already introduced the feign client for this in the ISTenantSyncListener class. I have missed removing this class from the PR. Did it with the latest commit